src/pages/About/Title.ts

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components/macro'
+import { BREAKPOINTS } from 'theme'
+
+export const Title = styled.h1<{ isDarkMode: boolean }>`
+  color: transparent;
+  font-size: 48px;
+  font-weight: 600;
+  margin-bottom: 0px;
+  background: ${({ isDarkMode }) =>
+    isDarkMode
+      ? 'linear-gradient(20deg, rgba(255, 244, 207, 1) 10%, rgba(255, 87, 218, 1) 100%)'
+      : 'linear-gradient(10deg, rgba(255,79,184,1) 0%, rgba(255,159,251,1) 100%)'};
+
+  background-clip: text;
+  -webkit-background-clip: text;
+
+  @media screen and (min-width: ${BREAKPOINTS.md}px) {
+    font-size: 72px;
+  }
+`
+
+export const SubTitle = styled.h2<{ isDarkMode?: boolean }>`
```

Comment:
Does figma specify a `ThemedText` variant here that we can use? If not, it might be worth nudging design to use theme styles instead

Reply:
this one is ridiculously specific unfortunately, and i feel it may a be a bit too late to change it out now